src/test/resources/test-home-dir/modules/lang-painless/plugin-security.policy

```diff
@@ -7,7 +7,7 @@
  * not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *    https://www.apache.org/licenses/LICENSE-2.0
+ *    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Please mind using `https` otherwise the build fails with our nohttp checks (Run `mvn … -Pci` locally to enable CI build verifications).

Reply:
fixed